### DISCUSSION TITLE
Issue with Budibase install on windows server

### DISCUSSION THREAD

**Discussion by mwblsz** [Budibase/budibase#4374] [Help]
I tried to install Budibase on a windows 2016 server, but keep getting the following error with docker-compose
"no matching manifest for windows/amd64"
I tried to add 
couchdb-service:
    platform: windows/amd64
to config file, but still got the same error.
Appreciate any help regarding this.

**Comment by shogunpurple** [comment]
hi @mwblsz - I get the feeling this may be down to some configuration in docker for windows. Here's a few things that you can try:

### Docker for windows settings
- Enable `"experimental": true`
- "Switch to Linux containers" option within docker for windows

The `ibmcom/couchdb3` image that is failing does not seem to have any images that support windows directly.
https://hub.docker.com/r/ibmcom/couchdb3/tags

**Reply by mwblsz** [reply]
thanks, I have enabled experimental, and as I read about it, Windows server 2016 does not support linux containers yet, here are the error I got now
PS C:\Users\Administrator\Budibase> docker-compose --env-file hosting.properties up
[+] Running 0/8
 - couch-init Error                                                                                                0.7s
 - watchtower-service Error                                                                                        0.7s
 - minio-service Pulling                                                                                           0.7s
 - proxy-service Error                                                                                             0.7s
 - couchdb-service Error                                                                                           0.7s
 - app-service Error                                                                                               0.7s
 - worker-service Error                                                                                            0.7s
 - redis-service Error                                                                                             0.7s
no matching manifest for windows/amd64 10.0.14393 in the manifest list entries
